app/controllers/studios_controller.rb

```diff
@@ -1,4 +1,7 @@
 class StudiosController < ApplicationController
+  include Pundit
+  after_action :verify_authorized, except: [:index, :show, :community], unless: :skip_pundit?
```

Comment:
in the LessonsController in the private you define skip_pundit, why is it not necessary here?